### DISCUSSION TITLE
What does LOGIN_REQUIRED do?

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#14699] [Q&A]
I have LOGIN_REQUIRED set to the default value of False.  According to the docs (quoted below), I should be able to access most data in Netbox.  But I don't see how to do that.  Any webpage I try to access redirect me to `/login`, and all the API endpoints (e.g. `/api/dcim/devices/` return 403 forbidden.  I tried changing this setting to True and I don't notice any difference in behavior.

> LOGIN_REQUIRED
> 
> Default: False
> 
> Setting this to True will permit only authenticated users to access any part of NetBox. By default, anonymous users are permitted to access most data in NetBox but not make any changes.
> 
>

**Comment by candlerb** [comment]
Netbox version?

**Reply by llamafilm** [reply]
3.7.0

**Reply by candlerb** [reply]
And do you have any additional authentication configured apart from the built-in usernames and passwords (e.g. LDAP, social-auth?)

**Reply by llamafilm** [reply]
Yes, I'm using social auth OIDC.  But I also tried disabling that and just using the default local login and the behavior is the same.

**Comment by markkuleinio** [comment]
Also set:

[EXEMPT_VIEW_PERMISSIONS](https://docs.netbox.dev/en/stable/configuration/security/#exempt_view_permissions) = ['*']
